scripts/rollup.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     format: 'cjs',
   },
   plugins: [
-    replace({ __DEV__: dev }),
+    replace({ __DEV__: dev, __SVG__: false }), // Needs attention! How do we decide when to turn it on?
```

Comment:
I would set this to true as the default, until we have a strategy there

Reply:
Will make the change. I saw found the flag false in the webpack config.